src/Network/GRPC/Client/Connection.hs

```diff
@@ -165,13 +166,18 @@ data ReconnectPolicy =
     -- connection), do not attempt to connect again.
     DontReconnect
 
-    -- | Reconnect after random delay after the IO action returns
+    -- | Reconnect to the (potentially different) server after the IO action
+    -- returns
```

Comment:
It might be worth explicitly pointing out that this can be used as a (poor-man's?) way to implement redundancy? Or are we worried that that particular man is _too_ poor? :thinking: